plugins/modules/cloud/misc/proxmox.py

```diff
@@ -214,6 +215,7 @@
     api_host: node1
     password: 123456
     hostname: example.org
+    storage: local-lvm
```

Comment:
Is there a specific reason to still have the storage option in this example (same for line 312)?

Reply:
@xilmen is there a reason you marked this as resolved without a comment?

Reply:
I add comment "Yes, In the first examples, we have specified "with minimal parameters", so for the following examples I have added it. This storage parameter blocking in some cases. Adding it, could help users to understand the errors they encounter."

Reply:
But I had not published it, sorry.